ui/StatusQ/src/StatusQ/Components/StatusMessage.qml

```diff
@@ -256,9 +256,10 @@ Control {
                     Loader {
                         Layout.fillWidth: true
                         active: (!root.editMode && !!root.messageDetails.messageText && !root.hideMessage
-                                 && ((root.messageDetails.contentType === StatusMessage.ContentType.Text)
-                                     || (root.messageDetails.contentType === StatusMessage.ContentType.Emoji) ||
-                                     (root.messageDetails.contentType === StatusMessage.ContentType.DiscordMessage)))
+                                 && ((root.messageDetails.contentType === StatusMessage.ContentType.Text) ||
```

Comment:
@caybro do we really need those conditions? I suppose we should show message text for every content type

Reply:
I think we do... there are message types (e.g. Image) where we don't want to render its text (which might still be used for a notification)